<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Typer AHK</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Enter text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="yourMessage">
        <property name="font">
         <font>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Typing time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="typeTime">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>15</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Time between lines:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="timeBetweenLine">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>2800</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Keypress Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="keyPressTime">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="cutLine">
          <property name="text">
           <string>Line cut:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineLimtChar">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>45</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reformatTextSmart">
          <property name="text">
           <string>Smart Full Reformat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fullReformatText">
          <property name="text">
           <string>Full Reformat Text</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="reformatText_2">
          <property name="text">
           <string>Reformat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="smartReformat">
          <property name="text">
           <string>Smart Reformat</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="dots">
        <property name="text">
         <string>add ... before new line
(only for smart reformat)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="customTime">
        <property name="text">
         <string>Generate custom time after...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="customSleepTime">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>550</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="AHKScreipt">
          <property name="text">
           <string>AHK Script:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeLabel">
          <property name="text">
           <string>Time: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="scriptAHK">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Activate Shortcut:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="shortcut">
          <property name="text">
           <string>Numpad9</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Chat key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="chatKey">
          <property name="text">
           <string>j</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Generate</string>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="saveRun">
          <property name="text">
           <string>Save and run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="genSaveRun">
          <property name="text">
           <string>Generate,  save and run</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="reformatText">
        <property name="text">
         <string>Reformat Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="everyWordInNewLine">
        <property name="text">
         <string>Every word in new line</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="AHKpath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pathSelect">
        <property name="text">
         <string>Select Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
